diesel_migrations/migrations_macros/Cargo.toml

```diff
@@ -24,3 +24,4 @@ proc-macro = true
 
 [features]
 default = []
+nightly = []
```

Comment:
We should not introduce a new feature flag here, as removing feature flags is quite hard (Would require a major version bump).
Also it seems like the corresponding rustc bug is "fixed" now, so maybe just using a newer nightly is sufficient?
https://github.com/rust-lang/rust/pull/71517